src/main/java/com/linkedin/replica/jobs/models/lightUser.java

```diff
@@ -1,12 +1,18 @@
-package models;
+package com.linkedin.replica.jobs.models;
+
+import com.arangodb.entity.DocumentField;
 
 public class lightUser {
+    @DocumentField(DocumentField.Type.KEY)
```

Comment:
Where to we use light models? If they are no longer needed please delete them. Also class names should be capitalized.